apps/website/src/content/docs/toast.mdx

```diff
@@ -1,18 +1,100 @@
 ---
 title: Toast
-description: A Toast is a non modal, unobtrusive window element used to display small amount of information to a user.
+description: A toast is a non-disruptive message that provides feedback based on a user action.
 thumbnail: Toast
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='toasts--positive' />
+## Usage
+
+Before triggering a toast, the toaster needs to be initialized using the `useToaster` hook. This hook returns an object which provides various methods for triggering the toast and changing the toaster settings.
 
 <LiveExample src='Toast.main.jsx'>
   <AllExamples.ToastMainExample client:load />
 </LiveExample>
 
```

Comment:
Would be good to mention this caveat, since it is a common issue that our users run into.

```suggestion

**Note**: `useToaster` must be called from the top level of a component which is rendered a descendant of [`ThemeProvider`](/docs/themeprovider).

```